app/api/predictions/controller.py

```diff
@@ -43,6 +43,7 @@ async def get_daily_event(
     # api_key: str = Depends(get_api_key)
                           ):
     try:
+        
```

Comment:
Probably you don't want this extra line.

Reply:
It should be uncommented, but Akash right now doesnt let us send information through the headers. I commented it so the FE can interact with the BE

Reply:
Oh, I mean line 46, the blank one.